src/status_im/ui/screens/chat/views.cljs

```diff
@@ -437,40 +438,46 @@
         (and
          mutual-contact-requests-enabled?
          one-to-one?
-         (not contact-added?))]
+         (not contact-added?))
+        loading-messages? @(re-frame/subscribe [:chats/loading-messages? chat-id])
+        all-loaded? @(re-frame/subscribe [:chats/all-loaded? chat-id])]
+
+    (if (or loading-messages? (not chat-id) (not all-loaded?))
```

Comment:
pls read discord, because we need to implement it in another way, move it to `list-footer` replacing
```
[react/view {:height 324 :align-items :center :justify-content :center}
        [react/activity-indicator {:animating true}]]
```

Reply:
Apologies @flexsurfer. 
I moved the skeleton to FlatList Footer. Please check.